arm_num: 70
arm_unit_length: 0.013
vendor_num: 2
arm_unit_radius: 0.02
linear_vel: 0.02
max_unit_angle_degree: 20